tutorials/fiori-ios-scpms-floorplan/fiori-ios-scpms-floorplan.md

```diff
@@ -470,7 +493,7 @@ private var products = [Product]()
 
 ```
 
-Also, we want to utilize the provided loading indicator. Let the class conform to the `SAPFioriLoadingIndicator` protocol.
+Also, we want to utilise the provided loading indicator. Let the class conform to the `SAPFioriLoadingIndicator` protocol.
```

Comment:
Please don't use the "royal we" unless you are referring to SAP. When giving instructions to the user, say "you" instead. Please also check the rest ot the tutorial and make necessary changes.